web/client/codechecker_client/cmd_line_client.py

```diff
@@ -1321,10 +1321,29 @@ def print_reports(client,
     if output_dir and not os.path.exists(output_dir):
         os.makedirs(output_dir)
 
+    if basename_local_dirs:
+        LOG.info("Matching local report directories (--baseline): %s",
+                 basename_local_dirs)
+    if newname_local_dirs:
+        LOG.info("Matching local report directories (--newname): %s",
+                 newname_local_dirs)
+
     client = None
     # We set up the client if we are not comparing two local report directory.
     if basename_run_names or newname_run_names:
-        client = setup_client(args.product_url)
+        try:
+            client = setup_client(args.product_url)
+        except SystemExit as sexit:
+            LOG.info("Could not connect to remote server uri=%s. "
+                     "Check if basenames and newnames are valid directory "
+                     "names or remote if accessible by URI.",
+                     args.product_url)
```

Comment:
Above we print matching local report directories which is useful information for the user. It would be also great to print the remote run names in this place too to identify that we are trying to fetch this information from the server. And if its failed we can print something like this to the ERROR log:

```suggestion
            LOG.error("Failed to get remote runs from server! Please "
                      "make sure that CodeChecker server is running on host "
                      "'%s' or these are valid directory paths!",
                      args.product_url)
```

Reply:
Done.